Formula/zookeeper.rb

```diff
@@ -34,14 +33,36 @@ def default_zk_env
     EOS
   end
 
-  def default_log4j_properties
+  def default_logback_xml
```

Comment:
Is there a resource we can download this from?

Reply:
it got customized from [this file](https://github.com/apache/zookeeper/blob/master/conf/logback.xml)

Reply:
I really don't think we should be making our own config. Let's just download that file instead.

Reply:
Sounds good, let me pull the sample one